### ISSUE TITLE
xkb include path failure

### ISSUE THREAD

**Issue by tsteven4** [MirServer/graphics-core22#7]
When I launch a Qt6 based app that uses QtWebEngine, the app fails with:
```
xkbcommon: ERROR: Couldn't find file "rules/evdev" in include paths
xkbcommon: ERROR: There are no include paths to search
xkbcommon: ERROR: 1 include paths could not be added:
xkbcommon: ERROR:       /usr/share/X11/xkb
xkbcommon: ERROR: Couldn't look up rules 'evdev', model 'pc101', layout 'us', variant '', options ''
[51723:51784:0522/165018.004909:FATAL:xkb_keyboard_layout_engine.cc(640)] Keymap file failed to load: us
```
This failure occurs regardless of the setting of XDG_CONFIG_ROOT.  I don't understand why XDG_CONFIG_ROOT is ignored.

In order to avoid this I have to symlink /usr/share/X11/xkb -> $SNAP/usr/share/X11/xkb
https://github.com/tsteven4/gpsbabel/blob/8533cc2a2f51d2f6255b6bfc3d4d8cd9d8cbd102/snap/snapcraft.yaml#L63-L66

Perhaps there is a better solution that relieves the user of graphics-core22 of this burden.

**Comment by AlanGriffiths** [comment]
The purpose of graphics-core22 is to provide an interface between application developers and providers of graphics userspace drivers. Keyboard layouts are not a good fit for this.

**Comment by tsteven4** [comment]
BTW this seems to be related to https://bugreports.qt.io/browse/QTBUG-105124:

- crash fixed in 6.5
- Setting the xkb config files path is not supported.
